autoPyTorch/pipeline/components/setup/traditional_ml/base_model.py

```diff
@@ -74,8 +80,10 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> autoPyTorchSetupComponent:
 
         # instantiate model
         self.model = self.build_model(input_shape=input_shape,
-                                      logger_port=X['logger_port'],
-                                      output_shape=output_shape)
+                                      logger_port=X['logger_port'] if 'logger_port' in X else None,
```

Comment:
I Think we should always have this here, the logger port

Reply:
I think in one of the tests we didn't have it